VMBackup/main/handle.py

```diff
@@ -277,6 +277,8 @@ def daemon():
             backup_logger.log(errMsg, True, 'Error')
 
         freezer = FsFreezer(patching= MyPatching, logger = backup_logger, hutil = hutil)
+        backup_logger.log(freezer.file_exists, True, 'Info')
```

Comment:
I think it might be better to not wait till daemon process to check file existence. This can be added as a basic validation just like we do for python install?

Reply:
As discussed offline adding it in a way similar to python validation would not be throwing an error, which would not be that useful in this case as it would be doing the validation in handle.sh file. So, going ahead with this.